frontends/ol-components/src/components/Banner/Banner.tsx

```diff
@@ -2,45 +2,83 @@ import React from "react"
 import styled from "@emotion/styled"
 import Typography from "@mui/material/Typography"
 import Container from "@mui/material/Container"
+import { ResponsiveStyleValue, SxProps } from "@mui/system"
+import { Theme } from "../ThemeProvider/ThemeProvider"
 
-const Description = styled(Typography)(({ theme }) => ({
+const SubHeader = styled(Typography)({
   maxWidth: "700px",
   marginTop: "8px",
-  [theme.breakpoints.down("sm")]: {
-    marginTop: "16px",
-  },
-}))
+})
 
 type BannerWrapperProps = {
   backgroundUrl: string
+  backgroundSize?: string
+  backgroundDim?: number
+  containerPadding?: string
 }
 
 /**
  * This is a full-width banner component that takes a background image URL.
  */
-const BannerWrapper = styled.div<BannerWrapperProps>(
-  ({ theme, backgroundUrl }) => ({
-    backgroundImage: `url(${backgroundUrl})`,
-    backgroundSize: "cover",
+const BannerWrapper = styled.header<BannerWrapperProps>(
+  ({
+    theme,
+    backgroundUrl,
+    backgroundSize = "cover",
+    backgroundDim = 0,
+    containerPadding = "48px 0 48px 0",
+  }) => ({
+    backgroundAttachment: "fixed",
+    backgroundImage: backgroundDim
+      ? `linear-gradient(rgba(0 0 0 / ${backgroundDim}%), rgba(0 0 0 / ${backgroundDim}%)), url('${backgroundUrl}')`
+      : `url(${backgroundUrl})`,
+    backgroundSize: backgroundSize,
     color: theme.custom.colors.white,
-    paddingTop: "48px",
-    paddingBottom: "48px",
+    padding: containerPadding,
     [theme.breakpoints.down("sm")]: {
-      paddingTop: "32px",
-      paddingBottom: "32px",
+      padding: "32px 0 32px 0",
     },
   }),
 )
 
-const NavText = styled(Typography)(({ theme }) => ({
-  color: theme.custom.colors.lightGray2,
-  marginBottom: "4px",
+const InnerContainer = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  alignItems: "flex-start",
+  justifyContent: "space-between",
+  [theme.breakpoints.down("md")]: {
+    flexDirection: "column",
+  },
+}))
+
+const HeaderContainer = styled.div({
+  display: "flex",
+  flexDirection: "column",
+})
+
+const RightContainer = styled.div(({ theme }) => ({
+  display: "flex",
+  flexDirection: "row",
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+  },
 }))
 
 type BannerProps = BannerWrapperProps & {
-  description: React.ReactNode
-  title: React.ReactNode
+  backgroundUrl: string
+  backgroundSize?: string
+  backgroundDim?: number
+  containerPadding?: string
   navText: React.ReactNode
+  avatar?: React.ReactNode
+  header: React.ReactNode
+  headerTypography?: ResponsiveStyleValue<string | undefined>
+  headerStyles?: SxProps<Theme>
+  subHeader?: React.ReactNode
+  subHeaderTypography?: ResponsiveStyleValue<string | undefined>
```

Comment:
"subheader" (and "subheading") are usually one word, so `subheader` w/o camel would be better.